tensorflow/tflite-micro
# Initial implementation of tool to generate integration tests.

Manually tested that tests can be generated with:
```
python tensorflow/lite/micro/integration_tests/generate_per_layer_tests.py --input_tflite_file tensorflow/lite/micro/models/person_detect.tflite --output_dir tensorflow/lite/micro/integration_tests/person_detect/conv
```
    
 And run with:
 ```
 make -f tensorflow/lite/micro/tools/make/Makefile test_integration_tests_person_detect_conv_test
 bazel test tensorflow/lite/micro/integration_tests/person_detect/conv:integration_test
 ```
    
 This tool is in very early stages of prototyping. Follow-up PRs will add tests and improve the tool.
    
 Some bugs:
     * http://b/205046520
     * http://b/205042963
     * http://b/204109200
    
 BUG=see description.
